pkg/backup/restorer/restorer.go

```diff
@@ -0,0 +1,110 @@
+package restorer
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/textileio/go-tableland/pkg/backup"
+)
+
+// BackupRestorer is responsible for restoring a database from a backup file.
+type BackupRestorer struct {
+	url, dst string
+}
+
+// NewBackupRestorer creates a new BackupRestorer.
+func NewBackupRestorer(url, dst string) *BackupRestorer {
+	return &BackupRestorer{
+		url: url,
+		dst: dst,
+	}
+}
+
+// Restore restores a database from a backup file URL.
+func (br *BackupRestorer) Restore() error {
```

Comment:
This is a very straightforward approach: download the file, decompress, and rename it. Lmk if this is good enough for now or we need to add extras